MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module(
+    name = "rules_pkg",
+    version = "0.6.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_python", version = "0.4.0")
+bazel_dep(name = "bazel_skylib", version = "1.0.3")
```

Comment:
Can we make this float to head, or whatever whomever depends on rules_pkg wants to use?

Reply:
Are we able to say we depend on bazel 5.x?